doc/api/util.md

```diff
@@ -660,18 +660,23 @@ Otherwise, returns `false`.
 See [`assert.deepStrictEqual()`][] for more information about deep strict
 equality.
 
-## util.promisify(original)
+## util.promisify(original[, options])
 <!-- YAML
 added: v8.0.0
 -->
 
 * `original` {Function}
+* `options` {Object} Optional. Can be passed to change the default behaviour.
```

Comment:
"Can be passed to change the default behavior." <- this part isn't necessary IMO and just adds noise.